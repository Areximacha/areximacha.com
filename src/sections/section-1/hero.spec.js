import React from 'react'
// import { render } from 'react-testing-library'
import renderer from 'react-test-renderer'

import Hero from './hero'

describe('<Hero/>', () => {
  describe('@render', () => {
    it('renders correctly', () => {
      const tree = renderer.create(<Hero />).toJSON()

      expect(tree).toMatchSnapshot()
    })
  })

  // describe('@events', () => {
  //   it('updates background translation on scroll', () => {
  //     const { getByTestId } = render(<Hero />)
  //   })
  // })
})
