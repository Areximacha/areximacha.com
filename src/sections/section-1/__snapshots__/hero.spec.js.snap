// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Hero/> @render renders correctly 1`] = `
<section
  className="css-g6bdmo"
  id="hero"
>
  <div
    className="css-7nheh8"
    data-testid="hero-bg"
  />
  <div
    className="css-ekjbro"
  >
    <h4>
      JAY
    </h4>
    <h4>
      TANG
    </h4>
  </div>
</section>
`;
